packages/mesos-modules/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -o nounset -o pipefail
```

Comment:
Don't need this / pkgpanda does it for you (please remove here, don't like copy / paste tech debt)